### ISSUE TITLE
[Feature Request] Option to overwrite core remaps or and/or manually create remaps not tied to game/core/directory

### ISSUE THREAD

**Issue by n1ghttrap** [libretro/RetroArch#14002]
It's pretty annoying to save a core remap only to decide you want to make a minor change, but have to recreate the whole thing after deleting the last one.  Thank you for your consideration.

**Comment by thingsiplay** [comment]
You can load the config, make changes, then delete and save a new one. So at least in the meantime don't delete the file before loading it.

**Comment by n1ghttrap** [comment]
No, that doesn't work.  It will load stock defaults for the core when you delete it, and will not retain any temporary settings you've made, so it's easy to lose your place.  When you delete you have to make it all over again.  An option to overwrite game/core would be greatly appreciated.

**Comment by thingsiplay** [comment]
It would be nice to have it off course. My suggestion was following: Load before deleting the file, so the current values are loaded up with the core. Then while it is loaded up, delete the file, make your changes in RetroArch and save it. I am not saying this is a good solution, just suggesting you a workaround until it is implemented.

**Comment by n1ghttrap** [comment]
It really isn't a solution.  Hoping it will be added in time.  Perhaps no one has considered it til now.

**Comment by Clydefrosch22** [comment]
why does the save remap button even disappear in the first place?

i keep running into this myself and it is so endlessly annoying especially when you're just feeling things out?

when i was still using retroarch on a 3ds, i swear i was able to overwrite the remaps over and over, why was this ever taken out?

**Comment by n1ghttrap** [comment]
This seems to work okay enough when you close out, and it makes changes to the game or core map when you do.  A manual save would be more assuring but this is effectively implemented.  Closing.
